src/events.rs

```diff
@@ -12,6 +12,7 @@ use tokio::sync::mpsc::Sender;
 use tokio::task::JoinHandle;
 
 pub type Notifier = Sender<Event>;
+pub type UserNotifier = Sender<UserNotifications>;
```

Comment:
Let's find more declarative names for this 2 senders. Maybe something like `SyncEventsSender` and `UserNotificationsSender`, what do you say?

Reply:
makes sense 👍